core/shared/src/main/scala/zio/stm/random/TRandom.scala

```diff
@@ -0,0 +1,260 @@
+package zio.stm.random
```

Comment:
Shouldn't this not be in the `random` package based on our new structure?

Reply:
Good catch :) On it.